drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -250,6 +250,10 @@ class RasterizerSceneGLES3 : public RendererSceneRender {
 		};
 
 		union {
+			struct {
+				uint64_t sort_key1;
```

Comment:
Will this initialize both keys, or just sort_key_1?

Reply:
[As far as I understand](https://github.com/godotengine/godot/pull/101344) it will initialize both keys:

> union initialization where only the very first element of the union is picked when using empty initializers such as {} if no other element in the union has an explicit initializer.

In this case the nameless struct is the first element in the union.

Reply:
Ahhh okay. I misunderstood what was causing the problem. I thought the first element of the struct was being grabbed, but no. The problem in these cases is that the first struct was smaller than the second. 

So a simple rule of thumb (that this PR enforces) is to always put the larger element first in a union